src/backend/joanie/core/models/products.py

```diff
@@ -756,13 +789,25 @@ def clean(self):
                 ]
                 if len(validated_user_orders) == 0:
                     message = _(
-                        f'Course run "{self.course_run.resource_link:s}" '
+                        f'Course run "{str(self.course_run.id)}" '
```

Comment:
There is a syntax for this in f-strings:
```suggestion
                        f'Course run "{self.course_run.id!s}" '
```